app/uk/gov/hmrc/helptosavefrontend/util/EmailVerificationParams.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend.util
+
+import java.util.Base64
+import javax.crypto.{Cipher, SecretKey, SecretKeyFactory}
+import javax.crypto.spec.DESKeySpec
+
+import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig._
+
+import scala.util.control.NonFatal
+
+case class EmailVerificationParams(nino: String, email: String) {
+  def encode(): String = {
+    val input = nino + ":" + email
```

Comment:
if we're going to be really anal about this it's safer to use a character like `§` to separate here and then split using that character when decoding since technically speaking emails may have colons in them. But the chance of someone having a colon in their email is probably remote enough for this not to matter too much.